### PR TITLE
added bunch of project pages

### PR DESCRIPTION
**Pull request by watakandai**
# Changes

I added a bunch of project pages.

# Work in progress

I need to add `title` tag

### CODE REVIEW

File: _projects/multiagent-planning.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 excerpt_separator: <!--more-->
-link-flag: false 
+link-flag: false
 bibkey: ma-planning
 ---
 
-As we begin employing teams of robots to accomplish complex tasks, it becomes difficult for a human to verify that future paths do not result in collisions. Therefore, we are working on new multiagent planning approaches that minimize the effot it takes a human to verify the paths. 
+As we begin employing teams of robots to accomplish complex tasks, it becomes difficult for a human to verify that future paths do not result in collisions. Therefore, we are working on new multiagent planning approaches that minimize the effot it takes a human to verify the paths.
 
 <!--more-->
```
Comment (Jmcjack) [inline _projects/multiagent-planning.md:17]: It looks like the rest of Justin's page is gone. I also need to figure out how to remove the content before  from the generated page.

Comment (watakandai) [reply _projects/multiagent-planning.md:17]: oh, I'll fix that.

Comment (watakandai) [reply _projects/multiagent-planning.md:17]: What do you mean by removing the content before from the generated page???

Comment (Jmcjack) [reply _projects/multiagent-planning.md:17]: The project pages are including the short summary at the top of the page. This isn't desired behavior, so we need to figure out how to suppress the text before the `<!--more-->` tag when generating the project page.

Comment (Jmcjack) [reply _projects/multiagent-planning.md:17]: I have a fix for this, so don't worry about it! 😁

Comment (watakandai) [reply _projects/multiagent-planning.md:17]: sounds good

**Pull request merged**